AkochB1Onboarding/app/src/main/java/com/example/akochb1onboarding/viewmodel/BlocksInfoSharedViewModel.kt

```diff
@@ -71,9 +48,12 @@ class MainActivityViewModel : ViewModel() {
         blockBatchFetchSemaphore = true
         viewModelScope.launch {
             val block = withContext(Dispatchers.IO) {
-                blockUseCase?.getBlock(blockId)
+                getLatestBlocksUseCase.getBlock(blockId)
```

Comment:
room and retrofit handles it's own queries by corroutines under the hood, so it's not recommendable to send this op to IO